internal/transactionsubmission/manager.go

```diff
@@ -274,5 +276,7 @@ func (m *Manager) loadReadyForProcessingBundles(ctx context.Context) ([]*store.C
 		return nil, fmt.Errorf("loading channel transaction bundles: %w", err)
 	}
 
+	log.Ctx(ctx).Infof("loaded %d bundles from the database", len(chTxBundles))
+
```

Comment:
Sorry for the misleading comment in https://github.com/stellar/stellar-disbursement-platform-backend/pull/21#discussion_r1309489856. When I said log, I actually meant "monitor" (my bad 😬).

```suggestion
```

I realized this is already being logged in:
https://github.com/stellar/stellar-disbursement-platform-backend/blob/dc974cf6c9104996e846a615e99a5f396704dfc1/internal/transactionsubmission/manager.go#L238

Could we add a monitor call after that line?

Reply:
I have to disagree that this needs a metric 🤔 I believe logs are enough to help us spot trends here unless there's a specific scenario that you want to be alerted on that I'm missing.

Reply:
I think being able to watch these numbers in a metrics aggregator is pretty useful, but considering you're already reporting processing/success/failure/reconciled payments, this one can be considered redundant, right?

So I agree we can skip this metric. Thanks for checking in with me 👍 

Just remember to remove this log, since there's already another log